backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -75,8 +75,12 @@ protected static Connection getDBConnection(RdbConfig rdb) throws SQLException {
   }
 
   protected static boolean checkDBConnection(RdbConfig rdb, Integer timeout) {
+    LOGGER.info("Checking DB connection with timeout [" + timeout + "]...");
     try (var con = getDBConnection(rdb)) {
-      return con.isValid(timeout);
+      LOGGER.info("Got connection: " + con);
```

Comment:
This seems at a glance like it would be better as DEBUG level, but I don't feel super strongly about it.

Reply:
yeah, probably a good idea. I think this happens only at startup time, so I don't know that it matters all that much, but I'm happy to downgrade it to debug. 👍🏼